### DISCUSSION TITLE
Should snap be able to install packages

### DISCUSSION THREAD

**Discussion by ttreptow** [kairos-io/kairos#1436] [Q&A]
I'm running kairos/ubuntu in aws based on the `quay.io/kairos/core-ubuntu-22-lts-rke2:v1.25.3-rke2r1` image. I noticed that when I log in, `apt` fails to install packages as expected, but I was able to `snap install` whatever I wanted. Is this expected behavior?

**Comment by mauromorales** [comment]
apt fails to install packages because of the nature of the immutable system. Could it be that snap is installing packages on your home directory?

**Comment by mudler** [comment]
Snapd persistence is supported by Kairos specifically. See here: https://github.com/kairos-io/kairos/pull/407
